.ropeproject/config.py

```diff
@@ -0,0 +1,112 @@
+# The default ``config.py``
+# flake8: noqa
+
+
+def set_prefs(prefs):
```

Comment:
is this somehow generally useful? maybe instead we should add `.ropeproject` to `.gitignore`

Reply:
I have no idea what this is! :confused:  Thinking related to ATOM I've added to ignore.